client/pages/home.js

```diff
@@ -29,6 +30,12 @@ export default withCampaignData(({campaigns, createCampaign}) => <ul>
 	<li>
 		<form onSubmit={createCampaign}>
 			<input placeholder='Campaign' name='title' />
+			<select name='calendar'>
```

Comment:
eventually this will want to move to a campaign settings page but it's fine here for now